docs/sine-points.html

```diff
@@ -3,9 +3,9 @@
 <body>
```

Comment:
I am not seeing sine-points render

Reply:
That would be fixed in improve-revision-checking (which I'm going to merge before release 0.7.0). But I made it here.